### PR TITLE
Update CI scripts

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

**Comment by dpo** [comment]
Thanks. Should we at least keep 1.3 since that's the minimum version [in Project.toml](https://github.com/JuliaSmoothOptimizers/Krylov.jl/blob/master/Project.toml#L13)?

**Comment by amontoison** [comment]
Yes, good idea.

**Comment by dpo** [comment]
Any idea why this errors out only on AMD? https://travis-ci.org/github/JuliaSmoothOptimizers/Krylov.jl/jobs/722902656#L558

**Comment by amontoison** [comment]
No :thinking: I modified the `check_min_norm` function a little bit to solve this problem.

File: test/check_min_norm.jl [hunk 1]
```diff
@@ -7,7 +7,8 @@ function check_min_norm(A, b, x; λ=0.0)
     AI = A
     xI = x
   end
-  xmin = AI' * ((AI * AI') \ b)
+  QR = qr(AI * AI')
```
Comment (dpo) [inline test/check_min_norm.jl:10]: Yes, the problem is probably that Julia tries to compute the LU of AA', which is singular. But we only need the QR of A', not of AA'.

Comment (amontoison) [reply test/check_min_norm.jl]: You're right !

**Comment by dpo** [comment]
Still failing. How about `x = Q * (R' \ b)`?

**Comment by amontoison** [comment]
I forgot to swap `R` and `Rᵀ`...
With `QR = Aᵀ`, `AAᵀ = RᵀR` and I should use the following code:
```julia
QR = qr(AI')
xmin = AI' * (QR.R \ (QR.R' \ b))
```

**Comment by dpo** [comment]
Sure, although multiplying by Q should be less prone to errors than another triangular solve and multiplying by A'.

**Comment by dpo** [comment]
🎉

**Pull request merged**